index.html

```diff
@@ -726,85 +822,43 @@ <h4>Automation via Workflows</h4>
-
-    </section>
-
-    <section>
-      <h3>Presenting</h3>
-      <p>
-The following APIs are defined for presenting a Verifiable Credential:
-      </p>
-
-      <table class="simple api-summary-table"
-        data-api-path="
-          /credentials/derive /presentations/prove
-          /presentations/available /presentations/submissions"
-        ></table>
-
       <section>
-        <h4>Derive Credential</h4>
-        <p>
+        <h4>Start Workflow</h4>
+        <p class="issue" title="Generalize 'Presentation Availability' endpoint">
+The existing <a href="#presentation-availability">Presentation Availability</a>
+endpoint requires a Verifiable Presentation to be submitted to the endpoint.
+The <em>Start Workflow</em> endpoint defined here generalizes  the <a
+href="#presentation-availability">Presentation Availability</a> endpoint by
+allowing arbitrary data to be posted to the endpoint. It is suggested that we
+can merge the API endpoint defined in this section with the <a
+href="#presentation-availability">Presentation Availability</a> endpoint via
+the generalization provided in this section. The term "workflow" is a working
+title and is not intended to be the final feature name.
         </p>
 
         <div class="api-detail"
-          data-api-endpoint="post /credentials/derive"></div>
+          data-api-endpoint="post /workflows/{name}/start"></div>
       </section>
 
       <section>
-        <h4>Prove Presentation</h4>
-        <p>
-        </p>
-
-        <div class="api-detail"
-          data-api-endpoint="post /presentations/prove"></div>
-      </section>
-
-      <section>
-        <h4>Presentation Availability</h4>
-        <p>
+        <h4>Workflow Interaction</h4>
+
+        <p class="issue" title="Generalize 'Submit Presentation' endpoint">
+The existing <a href="#submit-presentation">Submit Presentation</a> endpoint is
+highly similar to this endpoint and only differs in the return values provided
+by the endpoint. The <em>Workflow Interaction</em> endpoint defined here
+generalizes the  <a href="#submit-presentation">Submit Presentation</a> endpoint
+by allowing more variability in what is returned to the client and leaving some
+of the HTTP response codes undefined (which can be extended through other
+application-specific means). It is suggested that we can merge the API endpoint
+defined in this section with the <a href="#submit-presentation">Submit
+Presentation</a> endpoint via the generalization provided in this section. The
+term "workflow" is a working title and is not intended to be the final feature
+name.
         </p>
 
         <div class="api-detail"
-          data-api-endpoint="post /presentations/available"></div>
-      </section>
-
-      <section>
-        <h4>Submit Presentation</h4>
-        <p>
-        </p>
-
-        <div class="api-detail"
-          data-api-endpoint="post /presentations/submissions"></div>
+          data-api-endpoint="post  /workflows/{id}/presentations"></div>
```

Comment:
TODO: define the type of `id`.... UUID? URN? ... what about if `id` is a URL? lots of problems could be avoided by switching to a post body argument.

Reply:
this should definitely be defined, likely as a UUID

Reply:
yes, it MUST be defined if it is assumed to be in the route... it may be assumed to be a string, if it's in the post body.

Reply:
Agree that it needs to be defined more strictly. There is a tracking issue for where we put it here: https://github.com/w3c-ccg/vc-api/issues/256